### PR TITLE
1249 - Add a 'Rejected'/'Suitable' tag if an Assessment has been completed

### PR DESCRIPTION
**Pull request by richpjames**
# Context
PPs would like to be able to see if an assessment has been accepted or rejected rather than simply 'completed'

# Changes in this PR
If an assessment has been completed we check if it has a status of 'accepted' or 'rejected' and return the most suitable tag.

### Before
image

### After
Assess -- shows a list of assessments
ts of UI changes

### CODE REVIEW

File: server/utils/assessments/utils.ts [hunk 1]
```diff
@@ -334,7 +334,8 @@ const daysUntilDue = (assessment: Assessment): number => {
 
 const getStatus = (assessment: Assessment): string => {
   if (assessment.status === 'completed') {
-    return `<strong class="govuk-tag govuk-tag">Completed</strong>`
+    if (assessment.decision === 'accepted') return `<strong class="govuk-tag govuk-tag--green">Suitable</strong>`
```
Comment (pezholio) [inline server/utils/assessments/utils.ts:337]: Is "Suitable" definitely the wording we want here?

**Review by pezholio — APPROVED** [review]
LGTM - I'm assuming you've been your usual thorough self and checked that wording - I probably wouldn't have done it! 😆

File: server/utils/assessments/utils.ts (hunk 1 above)
Comment (richpjames) [reply server/utils/assessments/utils.ts:337]: Beth says she hasn't been able to go through this with Selina yet so the colours and copy are provisional for now.

Comment (richpjames) [reply server/utils/assessments/utils.ts:337]: Suitable was suggested in Slack: https://mojdt.slack.com/archives/C02SHKETZ7F/p1676460188359239 
But I think Accepted might make more sense

Comment (pezholio) [reply server/utils/assessments/utils.ts:337]: No drama - let's go with as is 👍

**Pull request merged**